python/ray/rllib/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 
 def _register_all():
-    for key in ["PPO", "ES", "DQN", "APEX", "A3C", "BC", "PG", "__fake",
+    for key in ["PPO", "ES", "DQN", "APEX", "A3C", "BC", "PG", "DDPG", "__fake",
```

Comment:
Could we rename this to "DDPG_baselines", and change the directory to `rllib/ddpg_baselines` in order to not conflict with the other DDPG pr here https://github.com/ray-project/ray/pull/1877/files ?

The main feature of this PR is that it's adapted from the baselines code, so we should probably keep that in the naming (or, could also call it `DDPG2` and just leave the details in the README).